index.html

```diff
@@ -1,23 +1,24 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"> 
+
```

Comment:
need to remove this whitespace because it triggers quirksmode in IE. First thing on line one should be the doctype declaration.